src/modules/QuickLinks/QuickLinks.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import classNames from 'classnames/bind';
+
+import { Button } from '@components/Button';
+import { Link } from '@components/Link';
+import { List, ListItem } from '@components/List';
+import { Paper } from '@components/Paper';
+import { SvgIcon, IconType } from '@components/SvgIcon';
+import { Typography } from '@components/Typography';
+
+import styles from './QuickLinks.module.css';
+import { QUICK_LINKS } from './constants';
+import { locale } from './locale';
+
+const cn = classNames.bind(styles);
+
+export function QuickLinks() {
+  return (
+    <Paper className={cn('quick-links')}>
+      <Typography className={cn('quick-links__heading')} variant="h3">
+        {locale.heading}
+      </Typography>
+      <List className={cn('quick-links__list')}>
+        {QUICK_LINKS.map((link) => {
+          return (
+            <ListItem className={cn('quick-links__list-item')} key={link.id}>
+              {link.action ? (
+                <Button disableRipple onClick={link.action} className={cn('quick-links__button')} noHover>
+                  <div className={cn('quick-links__button-content')}>
+                    <div>Connect Your Device</div>
```

Comment:
I think it should be {link.title}?

Reply:
for now we are assuming `Connect Your Device` is an Action button so only for the action I'm using button and if not action it is link